tsconfig.json

```diff
@@ -13,7 +13,8 @@
         "resolveJsonModule": true,
         "isolatedModules": true,
         "noEmit": true,
-        "jsx": "react"
+        "jsx": "react",
+        "noImplicitAny": false
```

Comment:
noImplicitAny: true, everything good otherweise

Reply:
Cool, I agree on this, but I just thought of handling it in a second time. This would avoid having to fix all implicit type errors at once

Reply:
great, but don't forget sinon ça va barder !

Reply:
au faite, ça build pas :D

Reply:
Fixed specifying the root for functions types:

```
"typeRoots": ["./node_modules/@types"]
```